sycl/test/esimd/genx_func_attr.cpp

```diff
@@ -16,24 +16,24 @@ __attribute__((sycl_kernel)) void kernel(Func kernelFunc) {
   kernelFunc();
 }
 
-SYCL_ESIMD_FUNCTION SYCL_EXTERNAL ESIMD_NOINLINE void callee(int x) {
```

Comment:
Why the parameter `x`  was removed here and on other functions?

Reply:
sorry i blindly copied this from [here](https://github.com/intel/llvm/pull/6462), probably something was different at that time. i added the argument back, but it is still optimized out due to this change.